README.md

```diff
@@ -9,10 +13,10 @@
 The intent of `Ard` is to be a modular, full-stack multi-disciplinary optimization tool for wind farms.
 
 The problem with wind farms is that they are complicated, multi-disciplinary objects.
```

Comment:
systems rather than objects, also "the problem" is a little much, maybe just start with "Wind farms are..."